src/Campaigns/Blocks/blocks.ts

```diff
@@ -1,18 +1,20 @@
 /**
  * WordPress dependencies
  */
-import {BlockConfiguration, getBlockType, registerBlockType} from '@wordpress/blocks';
+import {BlockConfiguration, getBlockType, registerBlockType, registerBlockVariation} from '@wordpress/blocks';
```

Comment:
Let's remove the unused `registerBlockVariation` import here.

Reply:
Resolved: 1c5d3f69583eb8741377952b2ecd0514fca29c61